examples/dashboard-operator/channels/packages/dashboard/1.8.3/manifest.yaml

```diff
@@ -165,3 +165,28 @@ spec:
       targetPort: 8443
   selector:
     k8s-app: kubernetes-dashboard
+
+
+---
+# ------------------- Application ------------------- #
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: kubernetes-dashboard
+  namespace: kube-system
+spec:
+  descriptor:
+    type: "kubernetes-dashboard"
+    description: "Kubernetes Dashboard is a general purpose, web-based UI for Kubernetes clusters. It allows users to manage applications running in the cluster and troubleshoot them, as well as manage the cluster itself."
+    icons:
+    - src: "https://github.com/kubernetes/kubernetes/raw/master/logo/logo.png"
+      type: "image/png"
+    maintainers:
+    - name: Jeff Johnson
```

Comment:
You might want to say here that this is a demo app, lest people think this is something you want people to run in production!

Reply:
excellent point, i'm dropping my name as well :)